app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    test = split_integer(16, 4)
+    assert sum(test) == 16
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    test = split_integer(6,2)
+    assert test == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    test = split_integer(8, 1)
```

Comment:
You can directly assert the result of `split_integer(8, 1)` without using the `test` variable to adhere to the checklist guidelines.